app/components/layout/menubar/login/Login.tsx

```diff
@@ -10,8 +10,12 @@ const Login = () => {
     return (
         <>
             {!casUserContext.currentUser.roles.includes(Role.USER)
-                ? <Button onClick={() => cas.login()}>Login</Button>
-                : <Button onClick={() => cas.logout()}>Logout ({casUserContext.currentUser.uid})</Button>}
+                ? <Button className="bg-bootstrap-button-blue" onClick={() => cas.login()}>
```

Comment:
Actually, could you make the color of the button the same as the h3 font? I think the class is text-gold. Thanks!